OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -88,7 +88,7 @@ public virtual void Tick(Actor self)
 		public bool FaceTarget(Actor self, Target target)
 		{
 			DesiredFacing = Util.GetFacing(target.CenterPosition - self.CenterPosition, TurretFacing);
-			return TurretFacing == DesiredFacing;
+			return Util.TickFacing(TurretFacing, DesiredFacing.Value, info.ROT) == DesiredFacing;
```

Comment:
I just tested it by setting TD turret `ROT` to `1`.
Didn't look to me as it got doubled `ROT` on this branch.